fuzz/config-fuzz.cpp

```diff
@@ -0,0 +1,14 @@
+#include <string_view>
```

Comment:
Is this a real error? If so, it may be prudent to notify the upstream maintainers - they could push a fix that would make it no longer necessary to catch the exception.